.phpcs.xml.dist

```diff
@@ -5,7 +5,9 @@
 	<rule ref="./.phpcs.config.xml" />
 
 	<!-- Default ruleset. -->
-	<rule ref="Jetpack" />
+	<rule ref="Jetpack">
+		<exclude-pattern>/projects/packages/analyzer/*</exclude-pattern>
```

Comment:
This should probably be changed for the changes in #22117.

Probably put it in `projects/packages/analyzer/.phpcs.dir.xml`.